src/vs/workbench/contrib/chat/browser/contrib/chatInputEditorContrib.ts

```diff
@@ -194,7 +194,31 @@ class InputEditorSlashCommandFollowups extends Disposable {
 	}
 }
 
-ChatWidget.CONTRIBS.push(InputEditorDecorations, InputEditorSlashCommandFollowups);
+class InputEditorSlashCommandAutoSend extends Disposable {
+	constructor(private readonly widget: IChatWidget) {
+		super();
+		this._register(this.widget.inputEditor.onDidChangeModelContent(() => this.onDidChangeModelContent()));
+	}
+
+	private onDidChangeModelContent() {
+		// If the input editor has a slash command that does not need additional args to run, submit it
+		const firstLine = this.widget.inputEditor.getModel()?.getLineContent(1);
```

Comment:
I think it feels unexpected to do this when I just typed out the command manually. Is there any way to know when it was selected from the suggest widget and just do it then?

If it's going to work this way, you should check that there is only one line of text

Reply:
I think we would need a new event from the completion provider to know that it was selected from the suggest widget, or maybe an optional command that is associated with a completion item and gets executed on selection, @jrieken any objections to me adding something like that?

Reply:
Nice, thanks both!